content/rc/security/access-control/saml-sso/_index.md

```diff
@@ -227,17 +227,33 @@ After you set up SAML SSO for one account, you can link other accounts you own t
 
 To link other accounts to an existing SAML SSO configuration:
 
-1. Go to **Access Management > Single Sign-On** in the Redis Cloud [admin console](https://app.redislabs.com).
+1. Sign in to the Redis Cloud [admin console](https://app.redislabs.com) with the account that has an existing SAML configuration.
 
-1. Select **Edit**.
+1. Go to **Access Management > Single Sign-On**. 
 
-1. For **Account linking**, select the checkboxes for the other accounts you want to link to SAML SSO.
+1. Select **Get token**.
 
-    {{<image filename="images/rc/access-management-saml-link-accounts.png"  alt="Link other accounts to the SAML configuration screen.">}}{{</image>}}
+    {{<image filename="images/rc/saml/popup-saml-get-token.png" alt="Get Token popup">}}{{</image>}}
 
-1. Select **Save**.
+    Select **Copy** to copy the linking token.
```

Comment:
Maybe change this line to a numbered step or combine it with the previous step like: "Select **Get token**, then copy the token."